### PR TITLE
CADC-11872, 11874: add session renew button, add session metadata to card

### PR DESCRIPTION
**Pull request by hjeeves**
Code changes:
CADC-11874: 
- new javascript function to handle renew request (science_portal_session.js)
- event handler for renew request (science_portal.js)
- new function button to renew session time frame (clock)
- error messaging for session list actions added - banner above session list shows errors for session actions (delete and renew).

CADC-11872:
- layout changes for SessionItem react component: buttons moved to footer, more metadata on card, header made smaller, button colours slightly different so they stand out a bit & match Launch button

Technical debt cleanup: 
- remove unneeded css, remove commented code, remove unused react app refs in science_portal_core functions

### CODE REVIEW

File: public/dev/js/science_portal.js [hunk 1]
```diff
@@ -260,14 +266,18 @@
...
-            "RAM": "2G",
-            "cores": "2",
+            "requestedRAM": this.requestedRAM,
+            "ramInUse": this.ramInUse,
+            "requestedCPUCores": this.requestedCPUCores,
+            "coresInUse": this.coresInUse,
             "altText": iconLabel,
             "connectURL": this.connectURL,
             "startTime": portalCore.dataFilters.startTime(this.startTime),
+            'expiryTime': portalCore.dataFilters.startTime(this.expiryTime),
```
Comment (at88mph) [inline public/dev/js/science_portal.js]: double quotes

File: public/dev/js/science_portal_session.js [hunk 2]
```diff
@@ -337,6 +338,48 @@
...
+            if (request.status === 200) {
+              resolve(sessionID)
+            } else {
+              reject(request)
+            }
+          },
+          false
+        )
+        // withCredentials enables cookies to be sent
+        // Note: SameSite cookie header isn't set with this method,
```
Comment (at88mph) [inline public/dev/js/science_portal_session.js]: Is this comment still relevant?  We set `SameSite: None` on our cookies now.

**Review by at88mph — CHANGES_REQUESTED** [review]

File: public/dev/js/science_portal.js (hunk 1 above)
Comment (hjeeves) [reply public/dev/js/science_portal.js]: Thanks. fixed.

File: public/dev/js/science_portal_session.js (hunk 2 above)
Comment (hjeeves) [reply public/dev/js/science_portal_session.js]: It's not relelvant, agreed. Removed 2 comments from this file.

**Review by at88mph — APPROVED** [review]

**Pull request merged**